iterm/verify-install.sh

```diff
@@ -0,0 +1,4 @@
+
+test -d $HOME/Applications/"iTerm.app"
+
+test -d $HOME/opt/homebrew-cask/Caskroom/iterm2/2.1.4/"iTerm.app"
```

Comment:
This line is cargo culted.  It was probably one of the first ones i made and should be changed to follow our other conventions.  (or maybe is needs to be here, but let's not be shy about moving removing things we think are unnecessary.